src/api.js

```diff
@@ -1446,6 +1453,7 @@ API.prototype.declare = function(options, handler) {
  *     accessToken:       '...'   // Access token for clientId
  *     // Client must have the 'auth:credentials' scope.
  *   },
+ *   raven:               null,   // raven.Client instance for error reporting
```

Comment:
but that would break my line width :)

Okay, probably we should make that clear...